e2e-tests/src/test/java/dev/streamx/cli/StreamxCliTestsIT.java

```diff
@@ -0,0 +1,143 @@
+package dev.streamx.cli;
+
+
+import dev.streamx.cli.test.tools.ResourcePathResolver;
+import dev.streamx.cli.test.tools.terminal.TerminalCommandRunner;
+import dev.streamx.cli.test.tools.validators.HttpValidator;
+import dev.streamx.cli.test.tools.validators.ProcessValidator;
+import io.quarkus.test.junit.QuarkusTest;
+import jakarta.inject.Inject;
+import jakarta.inject.Named;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+@QuarkusTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class StreamxCliTestsIT {
+
+  @ConfigProperty(name = "web.delivery.port.url", defaultValue = "http://localhost:8081/")
+  String webDeliveryPortUrl;
+
+  @Inject
+  @Named("StreamxCommandRunner")
+  TerminalCommandRunner terminalCommandRunner;
+
+  @Inject
+  ProcessValidator processValidator;
+
+  @Inject
+  ResourcePathResolver resourcePathResolver;
+
+  @Inject
+  HttpValidator httpValidator;
+
+  @BeforeAll
+  public void setup() {
+    streamxCommand(
+        "run -f " + filePath("streamx-mesh.yml"),
+        "STREAMX IS READY!",
+        60);
+  }
+
+  private String filePath(String fileName) {
+    return resourcePathResolver.absolutePath(fileName);
+  }
+
+  private void streamxCommand(String command, String expectedOutput, long timeoutInS) {
+    Process p = terminalCommandRunner.run(command);
+    processValidator.validateOutput(p, expectedOutput, timeoutInS * 1000);
+  }
+
+  @Test
```

Comment:
These tests looks like dupllicated.
Convert all tests to `ParameterizedTest`
(see example dev.streamx.cli.ingestion.publish.payload.JsonPathExtractorTest#shouldExtractPair)